### PR TITLE
Fixing an issue related to cmd window size in windows

### PR DESCRIPTION
**Pull request by yassingh**

### CODE REVIEW

File: hubblestack/files/hubblestack_nova/win_firewall.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def _get_tags(data):
 def _export_firewall():
     dump = []
     try:
-        temp = __salt__['cmd.run']('Get-NetFirewallProfile -PolicyStore ActiveStore', shell='powershell', python_shell=True)
+        temp = __salt__['cmd.run']('mode con:cols=1000 lines=1000; Get-NetFirewallProfile -PolicyStore ActiveStore', shell='powershell', python_shell=True)
```
Comment (yassingh) [inline hubblestack/files/hubblestack_nova/win_firewall.py]: following statement 
```temp = __salt__['cmd.run']('Get-NetFirewallProfile -PolicyStore ActiveStore', shell='powershell', python_shell=True)```
gives the output like
```
NotifyOnListen                  : False
EnableStealthModeForIPsec       : True
LogFileName                     : %systemroot%\system32\logfiles\firewall\domai
                                  nfw.log
LogMaxSizeKilobytes             : 16384
LogAllowed                      : True
LogBlocked                      : True
```

notice that the value of LogFileName is printed in two lines.  This causes failure in windows audit. 
```
 'Failure': [{'CIS-9.3.7': {'description': "(l1) ensure 'windows firewall - public - logging -  name' is set to '%systemroot%\\system32\\logfiles\\firewall\\publicfw.log'",
                            'found_value': '%systemroot%\\system32\\logfiles\\firewall\\publi',
                            'match_output': '%SYSTEMROOT%\\System32\\logfiles\\firewall\\publicfw.log',
                            'module': 'win_auditpol',
                            'name': 'LogFileName',
                            'nova_profile': 'windows-2012r2-level-1-scored-v2-2-1',
                            'tag': 'CIS-9.3.7',
                            'type': 'whitelist',
                            'value_type': 'public'}},
```

File: hubblestack/files/hubblestack_nova/win_firewall.py [hunk 2]
```diff
@@ -194,4 +194,4 @@ def _translate_value_type(current, value, evaluator, match):
         if match == '<':
             if int(current) < int(evaluator):
                 return True
-       return False
+    return False
```
Comment (yassingh) [inline hubblestack/files/hubblestack_nova/win_firewall.py]: Correcting the indentation.

**Comment by basepi** [comment]
Looks like we have a conflict. I'd also like @madchills to review.

**Comment by yassingh** [comment]
Resolved the confict.

**Pull request merged**